build-utils/download_and_unpack_jacoco.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+set -euxo pipefail
```

Comment:
Ist das der gleiche Fix wie Emanuel in vorgeschlagen hat oder ein anderer?

Reply:
Ja genau. Wir haben den Problematischen Befehl in dieses Skript ausgelagert. D.h. jede zukünftige Lösung bedarf nur die Anpassung dieser Datei und nicht aller Docker files.

Reply:
Ok, Danke